### PR TITLE
update octicons package to v8.x

### PR DESCRIPTION
**Pull request by shiftkey**
## Overview

This is just some tidy up of the latest release of the `octicons` package. `v8` looks like it has some new icons, and I clicked through the various screens of the app and couldn't spot anything unexpected.

## Release notes

Notes: no-notes

### CODE REVIEW

File: script/generate-octicons.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ async function generateIconData(): Promise<ReadonlyArray<IOcticonData>> {
 
   const results = new Array<IOcticonData>()
 
-  for (const name of Object.keys(octicons)) {
+  for (const name of Object.keys(octicons).sort()) {
```
Comment (shiftkey) [inline script/generate-octicons.ts]: I added this step to ensure the generated file is sorted alphabetically - the latest package has a different order after doing `const octicons = require('octicons')`

**Comment by outofambit** [comment]
@shiftkey does this help us with https://github.com/desktop/desktop/issues/5185 ? 🤔 What are the benefits of doing this upgrade?

**Comment by shiftkey** [comment]
@outofambit no, this is just a visual refresh of the current icons. I had a shot at migrating over to the new package some time ago and I think it didn't make a noticeable change in the generated bundle size, so I deferred it until I had some time to dig into how our bundles are generated.

**Comment by shiftkey** [comment]
Gonna merge this in while @outofambit looks into the work required to undertake #5185

**Pull request merged**